code/items/equipments/entities/C4Entity.cs

```diff
@@ -265,6 +265,17 @@ public static void PickUp(int c4EntityIdent, int playerIdent)
                 }
             }
         }
+
+        [ServerCmd]
+        public static void SetPreset(int c4EntityIdent, int preset)
+        {
+            var c4Entity = (C4Entity)FindByIndex(c4EntityIdent);
```

Comment:
1. Do not use `var`.
2. Pls do not cast directly, use the `is` keyword below. Otherwise, you could run into trouble with `(C4Entity) null`